### PR TITLE
Sources for the second dashboard demo

### PR DESCRIPTION
**Pull request by nuclearsandwich**
This demo isn't a package or network of packages but rather a set of build scripts and instructions for running the Dashboard demo using earthly.

The current form is capable of generating a build results archive but does not yet run the dashboard itself.

### CODE REVIEW

File: dashboard/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Add the [`process_sarif`](https://github.com/space-ros/process_sarif) project.
+```
+git clone https://github.com/space-ros/process_sarif spaceros_ws/src/process_sarif -b build-results-archive
+```
+
+Run a build and test
+
+```
+earthly +build-test +test
```
Comment (mjeronimo) [inline dashboard/README.md:18]: Just curious: wouldn't this command be the same as "earthly +test"? Does it just come down to style and which command is more clear about the intent?

**Review by mjeronimo — APPROVED** [review]

File: dashboard/README.md (hunk 1 above)
Comment (nuclearsandwich) [reply dashboard/README.md]: And in fact, earthly wouldn't even let me specify both. I was hoping that implicit vs explicit targets would change some behavior but as I can't specify multiple targets that is a moot point. I've changed it to just `earthly +test`

**Comment by ivanperez-keera** [comment]
Looking at this, without further context, I'd have no idea what this demo is about.

Should we add more information to the README?

**Comment by nuclearsandwich** [comment]
I would suggest closing this pull request as the demo build here is largely a copy-paste of the main spaceros build using earthly but with a single added step if incorporating the process_sarif project to generate the build-results-archive which is readable by the dashboard. I envision the replacement for this demo to be a guide which walks through generating the build-results-archive and opening it with the dashboard.

**Comment by ivanperez-keera** [comment]
Thanks, @nuclearsandwich .

Since we are preparing a release, I'm going to tag @Bckempa , @EzraBrooks and @mkhansenbot to see if any of this will be needed for the upcoming release. Otherwise, we can close the PR and revisit later as Steven suggested.

**Pull request closed without merging**